notebooks/pixiedust_facebook_analysis.ipynb

```diff
@@ -1856,9 +1855,12 @@
    "metadata": {},
    "source": [
     "# More Info.\n",
-    "For more information about PixieDust check out the following:\n",
-    "#### PixieDust Documentation: https://ibm-cds-labs.github.io/pixiedust/index.html\n",
-    "#### PixieDust GitHub Repo: https://github.com/ibm-cds-labs/pixiedust"
+    "For more information about PixieDust, check out the following:\n",
+    "* PixieDust Documentation: https://ibm-cds-labs.github.io/pixiedust/index.html\n",
+    "* PixieDust GitHub Repo: https://github.com/ibm-cds-labs/pixiedust\n",
+    "\n",
+    "Visit the Watson Accelerators portal to see more live patterns in action:\n",
+    "* Watson Accelerators: http://www.watsonaccelerators.com "
```

Comment:
normally we create the link hidden by using an actual hyperlink, but this is fine